src/pages/VenueLandingPage/VenueLandingPageContent.tsx

```diff
@@ -64,11 +66,18 @@ const VenueLandingPageContent: React.FC<VenueLandingPageContentProps> = ({
     if (!spaceSlug) return;
 
     const hasEntrance = world?.entrance?.length;
-
-    window.location.href =
-      user && !hasEntrance
-        ? generateAttendeeInsideUrl({ worldSlug: world?.slug, spaceSlug })
-        : venueEntranceUrl(world?.slug, spaceSlug);
+    const worldSlug = world.slug;
+
+    const redirectUrl = generateUrl({
+      route:
+        user && !hasEntrance
+          ? ATTENDEE_SPACE_INSIDE_URL
+          : ATTENDEE_SPACE_ENTRANCE_URL,
+      required: ["worldSlug", "spaceSlug"],
+      params: { worldSlug, spaceSlug, step: "1" },
+    });
+
```

Comment:
not sure, but we might need that same comment here as well
`/**
     * Using history.push() doesn't work correctly for some strange reason and firebase returns 403 (Missing or insufficient premissions).
     * It's almost like it doesn't detect when the user logs in and the rules consider the `uid` to be `null`
     */`

Reply:
debt comment has been moved to `VenuePage.tsx`